src/test/resources/slack-messages/expected-github-slack-request.json

```diff
@@ -1,3 +1,3 @@
 {
-  "text" : "\nHello team!\nThese pull requests have had no activity for more than two days:\n    *tdr-dev-documentation*\n         TomJKing: <https://github.com/nationalarchives/tdr-dev-documentation/pull/1|Add draft Terraform documentation> *Updated 747 days ago*\n    *tdr-prototype-mvc*\n         ian-hoyle: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/33|Add filters> *Updated 740 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/25|Add file upload page> *Updated 764 days ago*\n    *prototype-server*\n         ian-hoyle: <https://github.com/nationalarchives/prototype-server/pull/33|Add filters> *Updated 740 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/prototype-server/pull/25|Add file upload page> *Updated 764 days ago*\n\n"
+  "text" : "\nHello team!\nThese pull requests have had no activity for more than two days:\n    *tdr-dev-documentation*\n         TomJKing: <https://github.com/nationalarchives/tdr-dev-documentation/pull/1| Add draft Terraform documentation> Approved *Updated 747 days ago*\n    *tdr-prototype-mvc*\n         ian-hoyle: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/33| Add filters>  *Updated 740 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/25| Add file upload page>  *Updated 764 days ago*\n    *prototype-server*\n         ian-hoyle: <https://github.com/nationalarchives/prototype-server/pull/33| Add filters>  *Updated 740 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/prototype-server/pull/25| Add file upload page>  *Updated 764 days ago*\n\n"
 }
```

Comment:
Missing line break

Reply:
Ah no, I was getting so good at not forgetting them :smile: